spec/features/add_room_spec.rb

```diff
@@ -8,13 +8,9 @@
     within ".right" do
       click_on "Login with GitHub"
     end
-    click_link_or_button "Create a new room"
-
-    page.within("#new-room-form") do
-      expect(page).to have_content("Name")
-    end
 
     fill_in("room[name]", with: "Testing Suite")
+    fill_in("room[choice]", with: "rspec")
     click_on "Create room"
 
     page.within("#room-show") do
```

Comment:
If we are asserting that a room must be created with a choice, then it is not a different issue.